### ISSUE TITLE
Local image not showing

### ISSUE THREAD

**Issue by abergsneider** [tht13/html-preview-vscode#45]
**TL;DR:** Local image not showing

Hi all,

Currently attempting to test my html code with pictures found on my local computer.  After running say for example the line below:

`<p><img src="image_sample.PNG" alt="BaseFrequency" width="555" height="60"></p>`

There is no image shown in the preview, only **alt** name showing. I've tried toggling between various HTML security settings (found below) but not doing anything for me.

![image](https://user-images.githubusercontent.com/66797747/100156764-53a41d80-2e5e-11eb-994f-a964fc1097a3.png)

I'll be happy to provide any other detail if found necessary

**Some details about my system:**

### VS Code Version
Version: 1.51.1 (user setup)
Commit: e5a624b788d92b8d34d1392e4c4d9789406efe8f
Date: 2020-11-10T23:34:32.027Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.14393

### HTML Preview Version
Name: HTML Preview
Id: tht13.html-preview-vscode
Description: Provides ability to preview HTML documents.
Version: 0.2.5
Publisher: Thomas Haakon Townsend
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=tht13.html-preview-vscode

**Comment by maurocardinali** [comment]
Same problem here, any updates ?

**Comment by morningtundra** [comment]
Same issue on Mac OS 11.2.3.

Any update?

**Comment by Nostee** [comment]
The same issue right now.

**Comment by Ayanmullick** [comment]
I'm facing the same issue. The image loads fine on browsers but not on the preview in VSC.

**Comment by tqwewe** [comment]
I think I've figured out the issue...

Make sure your full path to your HTML file contains no spaces (`%20`)... having spaces seems to mess the images up.
